cfme/tests/cloud/test_tag_mapping.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+
+from cfme.cloud.provider.azure import AzureProvider
+from cfme.exceptions import ItemNotFound
+from cfme.markers.env_markers.provider import ONE_PER_CATEGORY
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.wait import wait_for
+
+from widgetastic.utils import partial_match
+
+pytestmark = [
+    pytest.mark.provider([AzureProvider], selector=ONE_PER_CATEGORY, scope='function'),
+    pytest.mark.usefixtures('setup_provider')
+]
+
+
+@pytest.fixture(scope='function')
+def map_tags(appliance, provider, request):
+    tag = appliance.collections.map_tags.create(entity_type=partial_match(provider.name.title()),
+                                                label='test',
+                                                category='Testing')
+    yield tag
+    request.addfinalizer(lambda: tag.delete())
+
+
+@pytest.fixture(scope='function')
+def vm(appliance, provider):
+    # cu-24x7 vm is tagged with test:testing in provider
+    tag_vm = provider.data.cap_and_util.capandu_vm
+    collection = provider.appliance.provider_based_collection(provider)
+    try:
+        return collection.instantiate(tag_vm, provider)
+    except IndexError:
+        raise ItemNotFound('VM for tag mapping not found!')
+
+
+@pytest.fixture(scope='function')
+def refresh_provider(provider):
+    provider.refresh_provider_relationships()
+    wait_for(provider.is_refreshed, func_kwargs={'refresh_delta': 10}, timeout=600)
+    return True
+
+
+def test_tag_mapping_azure_instances(vm, map_tags, refresh_provider):
+    """"
+    1. Find Instance that tagged with test:testing in Azure (cu-24x7)
```

Comment:
You can put this in the `Polarion` block under `testSteps` if you would like.  Enumerated list, there are examples in #8261